### PR TITLE
Lint positive #15

### PR DESCRIPTION
**Pull request by matq007**
Should be now lint positive and tests should pass.

### CODE REVIEW

**Comment by ewels** [comment]
Looking good! Though removing the pipeline test is _kinda_ cheating 😉 

What's the status on test data?

**Comment by matq007** [comment]
I've just commented them out for now. We have to make a smaller genome for a starter. I've managed to find another dataset from [fusioncatcher](https://github.com/ndaniel/fusioncatcher/tree/master/test). Ran tests and it looked good. Even STAR-Fusion returned 11 out of 16 fusions.
Requirements for fusioncatcher are at least 40GB for the human genome. So, either we somehow make a smaller genome or we drop down to another genome type. Any ideas, suggestions?

**Comment by ewels** [comment]
We've used Yeast and even PhiX in other pipelines. Not sure if these tools will run with those genomes though? Could be worth a shot?

**Comment by ewels** [comment]
Note that they don't have to actually _find_ any fusions. Just running is enough.

File: .travis.yml [hunk 1]
```diff
@@ -24,19 +22,17 @@ install:
...
+  - NXF_VER='0.32.0' # Specify a minimum NF version that should be tested and work
   - NXF_VER='' # Plus: get the latest NF version and check, that it works
 
 script:
   # Lint the pipeline code
   - nf-core lint ${TRAVIS_BUILD_DIR}
   # Run the pipeline with the test profile
-  - nextflow run ${TRAVIS_BUILD_DIR} -profile test,docker
+  # - nextflow run ${TRAVIS_BUILD_DIR} -profile test,docker --read 'data/reads_{1,2}.fq.gz' --genome GRCh38
+  - docker run -it -v ${TRAVIS_BUILD_DIR}:/${TRAVIS_BUILD_DIR} mproksik/rnafusion-test /bin/bash -c 'nextflow run ${TRAVIS_BUILD_DIR} -profile test --reads "/test-data/test_{1,2}.fastq.gz" --genome R64-1-1 --star_fusion'
```
Comment (MaxUlysse) [inline .travis.yml]: should `mproksik/rnafusion-test` be `nf-core/rnafusion-test` instead?

File: Dockerfile [hunk 2]
```diff
@@ -1,19 +1,20 @@
...
-    build-essential\
+    build-essential \
     libncurses5-dev \
     default-jdk \
     default-jre \
     gawk \
     gcc \
     g++ \
     bzip2 \
+    make \
```
Comment (MaxUlysse) [inline Dockerfile]: Is it normal to have `make` twice?
Personnaly, I sort the tools to install, that way, it's easier to spot duplicates

Comment (matq007) [reply Dockerfile]: Ahh, nice catch! 💪

File: .travis.yml (hunk 1 above)
Comment (ewels) [reply .travis.yml]: This is super cool 👍

**Comment by ewels** [comment]
> `Travis, work with me here`

...things going ok? 😆

**Comment by matq007** [comment]
fire

**Comment by matq007** [comment]
Aha! Who's the tough guy now Travis! **Do not merge yet, lint is commented out**.

**Comment by matq007** [comment]
Alright, few things that have to be done:
- [ ] Lint will cry about not pulling nfcore/rnafusion, can we change this to Warning when checking `.travis.yml`?
- [ ] I need someone to create a `rnafusion-test-dataset` container for me. The script is provided in `docker/rnafusion-test-dataset/build.sh`
- [ ] Run one more time to make sure everything works
- [ ] Rebase this madness 🤣 (small notes should be included in the commit or doc rearding build/travis?)
- [ ] Ship it 🚀

**Comment by matq007** [comment]
@ewels and @MaxUlysse you think we can merge this madness?

**Comment by MaxUlysse** [comment]
it's alive

Do you want me to squash it, or do you want to keep track of this madness?

**Comment by matq007** [comment]
Squash it, there is no need for having all of the commits. I'll make some notes afterward in the docs later on.

**Pull request merged**